sessions/react-global-state/demo-start/components/CounterStats.js

```diff
@@ -0,0 +1,22 @@
+const countAverage = "???";
+const countSum = "???";
+const dragonCount = "???";
+
+export default function CounterStats() {
```

Comment:
It might be easier to understand when we replace a local variable within the function with a prop from the parameters

```suggestion
export default function CounterStats() {
  const countAverage = "???";
  const countSum = "???";
  const dragonCount = "???";

```